ckb_dlfcn.h

```diff
@@ -413,7 +419,7 @@ int ckb_dlopen2(const uint8_t *dep_cell_hash, uint8_t hash_type,
         const char *current_str = shrtab + sh->sh_name;
         if (strcmp(DYNSTR, current_str) == 0) {
           const uint8_t *addr2 =
-              addr_offset_checked(aligned_addr, aligned_size, sh->sh_offset);
+              addr_offset_checked(aligned_addr, aligned_size, sh->sh_addr);
```

Comment:
Add reference or official documents about this changes.

Reply:
References mainly come from these places
In glibc, although sh_offset is used, it is read from the bin file here. In dlopen2, it reads from the memory that has been mapped.
And, it can be known from the official linux description that sh_addr is the address of the virtual memory, and sh_offset is the offset address of the binary file.

[glibc code](https://sourceware.org/git?p=glibc.git;a=blob;f=elf/sprof.c;h=3eaa4758d66c1293be1244a6b0cea313ad8e8146;hb=HEAD#l565)
[linux elf](https://refspecs.linuxfoundation.org/elf/elf.pdf) in page: 1-11
[Wiki linux elf](https://en.wikipedia.org/wiki/Executable_and_Linkable_Format)